infrastructure/metadata/infrastructure/server/webtransport-h3.sub.any.js.ini

```diff
@@ -0,0 +1,24 @@
+[webtransport-h3.sub.any.html]
+  expected: [OK, FAIL, ERROR]  # TODO(bashi): Enable the WebTransport test server and fix https://crbug.com/1237781. Ditto below.
```

Comment:
It's surprising to me that we're marking this as (effectively) intermittent, and in particular `FAIL` isn't a status we can get here. It would make more sense to me if we had specific per-product statuses which we can get using e.g. (not the actual results, because I don't know what those are)

```
[webtransport-h3.sub.any.html]
  expected:
    if product == "chrome": OK
    ERROR
  [WebTransport server should be running and should handle a bidirectional stream]
    expected:
      if product == "safari": [FAIL, ERROR]
      if product == "firefox": ERROR
```

Reply:
Thank you for the suggestion!

.ini expectations really puzzled me. I feel that I still don't understand how it works and what you're suggesting (what's the difference between FAIL and ERROR, what's are expectations for nested items, do I need or don't need .window or .worker etc). I tried your suggestion but `wpt run` existed with status code 1.

Reply:
I suspect this was an attempt to deal with the apparent flakiness in Chrome, which has been fixed. @bashi since the test fails everywhere now, perhaps this can be simplified? It should be the same for all browsers until support is enabled in Chrome.

Reply:
Thank you @foolip for the help! I updated the expectation. Does it look reasonable?

Reply:
Almost! I left a new review comment suggesting some further simplification :)

Reply:
To give some detail about the expectation data:
* `FAIL` means an assert in the test failed. `ERROR` means that there was an exception in the test not related to an assert.
* The difference between the nested items and the outer items is that the outer items represent the test file and the nested item represent specific subtests within the file. So if you have a testharness test with 10 subtests and 5 pass and 5 fail, you'll have 5 nested items, one for each failing subtest (`PASS` is the default status). Obviously for the case where there's only one subtest that looks like redundant information.
* There is one ini file per test file, but each test file can have more than one associated test (e.g. with `.any.js`). Each test with non-default statuses requires a seperate item since they can be different (e.g. in this case the `sharedworker` variant fails only in WebKit).

There's some (somewhat outdated) documentation for this at https://web-platform-tests.org/tools/wptrunner/docs/expectation.html

In regard to automatic updates, `wpt update-expectations --product=chrome wptreport.json` should work (gecko uses this a lot internally), but we don't currently have a way to get it to work with mutliple products and automatically generate per-product conditionals (this is just poor factoring rather than anything difficult; we can generate conditionals for different configuations of a single product but can't handle using product itself as the differentiating factor).